cloudflare/resource_cloudflare_worker_script.go

```diff
@@ -27,11 +27,36 @@ func resourceCloudflareWorkerScript() *schema.Resource {
 				Required: true,
 				ForceNew: true,
 			},
-
 			"content": {
 				Type:     schema.TypeString,
 				Required: true,
 			},
+			"binding": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"name": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"kv_namespace_id": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"plain_text": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"secret_text": {
+							Type:      schema.TypeString,
+							Sensitive: true,
+							Optional:  true,
+						},
```

Comment:
I wanted to use `ExactlyOneOf` schema check here but couldn't get it to work. Tried to search for examples in other providers like AWS but also see they only use it for the top-level schema and not within a `TypeSet`. Let me know if there's a simple way to achieve this validation, otherwise we'll be relying on documentation.

Reply:
`Set`s are pretty annoying to work with for these cases. I think we start with documentation and see how we go before spending too much time on solving it technically.